src/components/v2/V2Project/NewDeployModal.tsx

```diff
@@ -56,13 +56,13 @@ export default function NewDeployModal({
       cancelText={t`Later`}
     >
       <h2>
-        <Trans>Next steps</Trans>
+        <Trans>Next steps (optional)</Trans>
       </h2>
       <p>
         <Trans>
-          Congratulations on launching your project! We recommend you take these
-          next steps to help your project gain traction. These steps are
-          optional and you can complete them later on.
+          Congratulations on launching your project! The following steps are{' '}
```

Comment:
I think this sentence might be a bit clunky, I wonder if it is better as:
> The following steps are **completely optional** and can be completed at any time.

Not sure if we need to tell the user that they are useful as they can probably make the decision themselves, wdyt?

Reply:
Yeah good call, I back it